.github/workflows/build-php-table-backend-utils.yml

```diff
@@ -52,51 +52,73 @@ env:
 
-defaults:
-  run:
-    working-directory: packages/php-table-backend-utils
-
 jobs:
   build_image_php74_table_backend_utils:
     if: ${{ inputs.hasCodeChanged || inputs.isTag }}
     runs-on: ubuntu-latest
     steps:
       - name: Checkout
         uses: actions/checkout@v3
+
+      - name: Download artifact
+        uses: actions/download-artifact@v3
+        with:
+          name: ci-monorepo
+          path: /tmp
+      - name: Load Base monorepo Docker image
+        run: |
+          docker load --input /tmp/ci-monorepo.tar
+      - name: Monorepo Localize Composer Paths
+        run: |
+          docker-compose run ci-monorepo composer install
+#          docker-compose run ci-monorepo vendor/bin/monorepo-builder localize-composer-paths packages/php-table-backend-utils/composer.json --ansi
+
       - name: Build docker image
         env:
           AWS_ACCESS_KEY_ID: ${{ inputs.BUILD_AWS_ACCESS_KEY_ID }}
           AWS_SECRET_ACCESS_KEY: ${{ secrets.BUILD_AWS_SECRET_ACCESS_KEY }}
         run: |
-          docker-compose build production74
+          docker-compose build ci-php-table-backend-utils74
       - name: Check
         run: |
-          docker-compose run production74 composer check
+          docker-compose run ci-php-table-backend-utils74 composer check
   build_image_php_table_backend_utils:
     if: ${{ inputs.hasCodeChanged || inputs.isTag }}
     runs-on: ubuntu-latest
     steps:
-      -
-        name: Checkout
+      - name: Checkout
         uses: actions/checkout@v3
+
+      - name: Download artifact
+        uses: actions/download-artifact@v3
+        with:
+          name: ci-monorepo
+          path: /tmp
+      - name: Load Base monorepo Docker image
+        run: |
+          docker load --input /tmp/ci-monorepo.tar
       -
-        name: Build docker image
-        env:
-          AWS_ACCESS_KEY_ID: ${{ inputs.BUILD_AWS_ACCESS_KEY_ID }}
-          AWS_SECRET_ACCESS_KEY: ${{ secrets.BUILD_AWS_SECRET_ACCESS_KEY }}
+        name: Monorepo Localize Composer Paths
         run: |
-          docker-compose build production
-          docker save -o /tmp/php-table-backend-utils.tar table-utils:latest
+          docker-compose run ci-monorepo composer install
+#          docker-compose run ci-monorepo vendor/bin/monorepo-builder localize-composer-paths packages/php-table-backend-utils/composer.json --ansi
       -
         name: Check
+        env:
+          DOCKER_BUILDKIT: 1
+          AWS_ACCESS_KEY_ID: ${{ inputs.BUILD_AWS_ACCESS_KEY_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.BUILD_AWS_SECRET_ACCESS_KEY }}
         run: |
-          docker-compose run production composer check
+          docker-compose build ci-php-table-backend-utils
+          docker-compose run ci-php-table-backend-utils composer check
+          docker commit $(docker ps -alq) ci-php-table-backend-utils:latest
+          docker save -o /tmp/ci-php-table-backend-utils.tar ci-php-table-backend-utils:latest
```

Comment:
kdybys to nepřepisoval někde na cache, tak by bylo fajn abys to ukládal jinam než přímo do /tmp IMHO. V /tmp se může ocitnout spousta různých věcí... a pak někde uploadneme celý temp...